util/libcrypto.num

```diff
@@ -4389,4 +4389,3 @@ EVP_aria_256_ccm                        4332	1_1_1	EXIST::FUNCTION:ARIA
 EVP_aria_128_gcm                        4333	1_1_1	EXIST::FUNCTION:ARIA
 EVP_aria_128_ccm                        4334	1_1_1	EXIST::FUNCTION:ARIA
 EVP_aria_192_gcm                        4335	1_1_1	EXIST::FUNCTION:ARIA
-CRYPTO_THREAD_glock_new                 4336	1_1_1	EXIST::FUNCTION:
```

Comment:
Good! Last time around I didn't manage to get to this in time. I only managed to ask what about locks allocated by libssl, got answer that they are not global, but didn't manage to ask in time why this function is exported then...

Reply:
Just in case, even though I suggested to replace referencing array with function call (but I don't feel strongly about it) I definitely didn't mean to suggest to export it...

Reply:
I definitely don't want to export it, and make sure that we can add more in the future if we need to!